homeassistant/components/websocket_api/http.py

```diff
@@ -302,22 +301,22 @@ def handle_hass_stop(event: Event) -> None:
                 raise Disconnect
 
             try:
-                msg_data = msg.json(loads=json_loads)
```

Comment:
This was typed to Any so we lost typing when go through here, but `.data` is available (as used in the binary handler)